.github/workflows/update-gcs-bucket.yaml

```diff
@@ -9,39 +9,62 @@ on:
 
 env:
   node_version: 16
+  typesense_url: https://typesense.camino.network
+  default_collection_name: camino-docs-default
 
 jobs:
-  BuildAndDeploy:
+  build_deploy_scrape:
+    name: Build, Deploy & Scrape
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v3
-      - uses: actions/setup-node@v3
-        with:
-          node-version: ${{ env.node_version }}
-          
-      - name: install yarn dependencies        
-        run: yarn install
 
-      - name: build
-        run: yarn build
-
-      - name: set bucket names
-        id: setBucketNames
+      - name: Prepare variables
+        id: prepareVariables
         run: |
           if [[ "${{ github.ref_name }}" == "main" ]]; then
               echo "bucket=docs.camino.network" >> $GITHUB_OUTPUT
+              echo "collection=camino-docs-`date +%s`" >> $GITHUB_OUTPUT
+              echo "loadbalancer=docs-lb" >> $GITHUB_OUTPUT
```

Comment:
what's the loadbalancer variable used for ?

Reply:
I was going to use it for cache invalidation but then decided to ditch that idea as we already account for old and new collections in typesense. Left the variable there but I think it's just confusing now. I can remove it.

Reply:
yes pls. remove

Reply:
Done